ntoskrnl/mm/ARM3/procsup.c

```diff
@@ -890,6 +901,15 @@ MiInsertSharedUserPageVad(VOID)
         return Status;
     }
 
+    Status = PsChargeProcessNonPagedPoolQuota(PsGetCurrentProcess(), sizeof(MMVAD_LONG));
```

Comment:
Use the `Process` argument that is being given by the caller of `MiInsertSharedUserPageVad`.